go.sum

```diff
@@ -20,6 +20,8 @@ github.com/blang/semver v3.5.1+incompatible/go.mod h1:kRBLl5iJ+tD4TcOOxsy/0fnweb
 github.com/bsm/go-vlq v0.0.0-20150828105119-ec6e8d4f5f4e/go.mod h1:N+BjUcTjSxc2mtRGSCPsat1kze3CUtvJN3/jTXlp29k=
 github.com/cedexis/go-itm v1.0.0 h1:tcnLi0u0xatMsJ74LrNWuFfiyqxShIoq7mas22bpbxs=
 github.com/cedexis/go-itm v1.0.0/go.mod h1:E62ZBgPOgT5r1dHBBPjh2xhL8JXM5glyxG80K8xCJK4=
+github.com/cedexis/go-itm v1.0.1 h1:YMkMHsbfpXCcO7JC4I/va9tKhqhnRg6sUbk8K5JTjRk=
+github.com/cedexis/go-itm v1.0.1/go.mod h1:E62ZBgPOgT5r1dHBBPjh2xhL8JXM5glyxG80K8xCJK4=
```

Comment:
Weird, it doesn't remove the other `go-itm` entry in the `go.sum`?  I assumed it'd be like other lock file formats that remove unreferenced versions from the file. Well, this system is pretty new to them, maybe they'll improve that with time.

Reply:
Yeah seems weird. But apparently by design...

https://github.com/golang/go/wiki/Modules#is-gosum-a-lock-file-why-does-gosum-include-information-for-module-versions-i-am-no-longer-using